packages/aws-cdk-lib/aws-ecs/test/base-service.test.ts

```diff
@@ -82,7 +82,7 @@ describe('For alarm-based rollbacks', () => {
     stack = new cdk.Stack();
   });
 
-  test('deploymentAlarms is set by default for ECS deployment controller', () => {
+  test('deploymentAlarms is NOT set by default for ECS deployment controller', () => {
```

Comment:
This is a breaking change. We need a feature flag for this.

Reply:
Added a feature flag.